packages/components/MenuButton/macos/MenuButton.swift

```diff
@@ -0,0 +1,111 @@
+//
```

Comment:
I don't think this matters in terms of functionality but for my own curiosity, what's the reason to write the manager in obj-c and the menu button in swift?

Reply:
The Manager must be written in obj-c because it uses Macros to talk to React Native. The View itself however can be written in Swift with @objc() wrappers around appropriate properties and methods. React Native docs describe how to this here: https://reactnative.dev/docs/native-modules-ios#exporting-swift

I also personally try to use as much Swift as possible, since that's the newer / nicer / safer language to use.